tpot/base.py

```diff
@@ -563,12 +581,17 @@ def score(self, testing_features, testing_classes):
         if self._fitted_pipeline is None:
             raise RuntimeError('A pipeline has not yet been optimized. Please call fit() first.')
 
-        # If the scoring function is a string, we must adjust to use the sklearn scoring interface
-        return abs(SCORERS[self.scoring_function](self._fitted_pipeline,
-            testing_features.astype(np.float64), testing_classes.astype(np.float64)))
+        # If the scoring function is a string, we must adjust to use the sklearn
+        # scoring interface
+        score = SCORERS[self.scoring_function](
+            self._fitted_pipeline,
+            testing_features.astype(np.float64),
+            testing_classes.astype(np.float64)
+        )
+        return abs(score)
```

Comment:
Is returning `abs` of the score the correct thing to do here? The blame graph shows that we added it when we added regressors, but what is the thinking behind it?